src/test/kotlin/racingcar/RaceManagerTest.kt

```diff
@@ -1,20 +1,45 @@
 package racingcar
 
 import io.kotest.matchers.shouldBe
+import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
 import racingcar.domain.CarFactory
 import racingcar.domain.RaceManager
+import racingcar.domain.checker.RandomRaceConditionChecker
+import racingcar.fake.FakeRaceConditionChecker
 
 class RaceManagerTest {
     @Test
     fun `자동차 경주 우승자 이름을 얻을 수 있다`() {
         val cars = CarFactory.generateCars(listOf("pobi", "crong", "honux"))
         val raceManager = RaceManager(moveCount = 5, cars = cars)
 
-        val raceHistories = raceManager.startRacing()
-        val finalRaceHistory = raceHistories.last()
-        val maxPosition = finalRaceHistory.cars.maxByOrNull { it.position }?.position
-        val champions = finalRaceHistory.cars.filter { it.position == maxPosition }.map { it.name }
-        finalRaceHistory.findRacingChampions() shouldBe champions
+        val raceResult = raceManager.startRacing(RandomRaceConditionChecker())
+
+        raceResult.findRacingChampions() shouldBe raceResult.raceHistories.last().findMaxPositionCarNames()
```

Comment:
`findRacingChampions` 메서드가 결국 내부적으로 `raceResult.raceHistories.last().findMaxPositionCarNames()`를 호출하고있기 때문에 이 테스트 케이스는 검증 로직이 별로 의미가 없는 것 같아요. 🤔

의도한 검증 결과가 잘 나오는지 아래와 같이 표현하는게 훨씬 더 직관적이라고 생각되네요. 😃
```suggestion
        raceResult.findRacingChampions() shouldBe listOf("pobi", "crong", "honux")
```

Reply:
작성하면서 애매하다고 생각했던 부분인데 잘 캐치해주셔서 감사합니다!